app/templates/components/post-header.hbs

```diff
@@ -1,2 +1,2 @@
 <div class="post-icon"></div>
-{{post-title post=post}}
+{{post-title post=post saveTitle=saveTitle}}
```

Comment:
For me it's confusing, maybe I'm just too new to all this. For instance, if I see `{{post-header save=(route-action 'save'}}`, `{{post-title save=save}}`, and in the 'post-title' component, I see `<button {{action 'save'}}`. Then in `post-title.js` I see 
```javascript
save() {
  this.sendAction('save', post);
}
```
I really lose my bearings.